src/main/java/com/icthh/xm/tmf/ms/resource/security/SecurityUtils.java

```diff
@@ -1,10 +1,11 @@
 package com.icthh.xm.tmf.ms.resource.security;
 
-import java.util.Optional;
 import org.springframework.security.core.context.SecurityContext;
 import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.security.core.userdetails.UserDetails;
 
+import java.util.Optional;
+
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version